cmd/tikv-ctl/src/main.rs

```diff
@@ -687,16 +711,26 @@ fn print_bad_ssts(db: &str, manifest: Option<&str>, pd_client: RpcClient, cfg: &
         // The corruption format may like this:
         // /path/to/db/057155.sst is corrupted: Corruption: block checksum mismatch: expected 3754995957, got 708533950  in /path/to/db/057155.sst offset 3126049 size 22724
         println!("corruption info:\n{}", line);
```

Comment:
Does it includes empty line? Maybe we should skip those up front with `line.trim().is_empty()`

Reply:
I remember the pd-client logs are included, can we solve it?

Reply:
Don't know. Maybe they are not processed here. The lines here appears to be taken from the "child_process".

Reply:
"I remember the pd-client logs are included, can we solve it?" This is for @hunterlxt

Reply:
Maybe we should set tikv-ctl log level to warning, what's your suggestion?

Reply:
> Does it includes empty line? Maybe we should skip those up front with `line.trim().is_empty()`

It will be skipped below
```
        let r = Regex::new(r"/\w*\.sst").unwrap();
        let sst_file_number = match r.captures(line) {
            None => {
                continue;
            }
```

Reply:
> "I remember the pd-client logs are included, can we solve it?" This is for @hunterlxt

https://github.com/tikv/tikv/pull/11657 deprecated `log_level` but forget to modify logic in tikv-ctl